tests/integration/cli/test_cli.py

```diff
@@ -21,7 +21,6 @@
         "data_dir": "tests/assets/classification_dataset",
         "overrides": [
             "model.otx_model.config.head.num_classes=2",
-            "model.otx_model.config.data_preprocessor.num_classes=2",
```

Comment:
Actually, DINOv2 doesn't use the data_preprocessor.num_classes..

Reply:
Since we discussed before, num_classes for the data_preprocessor will not affects to the other models. So, I think removing this line is better. We couldn't use the `data_preprocessor.num_classes` field if we want to use non mmbased model.

Reply:
However, since this override is used by other models as well, I don't think it makes sense to remove it for DINOv2 only. Even if it doesn't use in DINOv2 actually, it would be nice to keep the `data_preprocessor.num_classes` in to config format it with other classification configs.

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2708/commits/94282b0015747affa828f916615f4f9d0bd36de3